makefiles/Makefile.docs-node

```diff
@@ -23,28 +22,19 @@ help: ## Show this help message
 	@echo 'Variables'
 	@printf "  \033[36m%-18s\033[0m %s\n" 'ARGS' 'Arguments to pass to mut-publish'
 
-next-gen-html-publish:
-	# snooty parse and then build-front-end
+download-published-branches:
 	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-node.yaml > ${REPO_DIR}/published-branches.yaml
-	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}"; \
-	if [ $$? -eq 1 ]; then \
-		exit 1; \
-	else \
-		exit 0; \
-	fi
-	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR};
-	curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/configfiles/docs-node/gatsby-config.js > ${REPO_DIR}/snooty/gatsby-config.js
-	cd snooty; \
-	echo "GATSBY_SITE=${PROJECT}" > .env.production; \
-	echo "GATSBY_PARSER_USER=${USER}" >> .env.production; \
-	echo "GATSBY_PARSER_BRANCH=${GIT_BRANCH}" >> .env.production; \
-	echo "COMMIT_HASH=${COMMIT_HASH}" >> .env.production; \
-	@npm run build; \
-	cp -r "${REPO_DIR}/snooty/public" ${REPO_DIR};
-	
-next-gen-publish: next-gen-html-publish
+
+configure-mut-redirects:
 	if [ ${GIT_BRANCH} = master ]; then mut-redirects config/redirects -o public/.htaccess; fi
-	
+
+deploy-search-index: ## Update the search index for this branch
+	@echo "Building search index"
+	mut-index upload build/public -o ${PROJECT}-${GIT_BRANCH}.json -u ${PRODUCTION_URL}/${PROJECT} -g -s --exclude build/public/sdk/iOS
+
+%:
+	@:
+
 next-gen-html:
```

Comment:
Doesn't this target have a dependency on the new `download-published-branches` target?

Reply:
we would actually run each target separately, see:https://github.com/madelinezec/docs-worker-pool/blob/DOP-1238/worker/jobTypes/githubJob.js#L275-L276

Reply:
Interesting, what is the reasoning for that?

Reply:
It can be very difficult to call one makefile target from another target

Reply:
And in this case, you can't have next-gen-publish: download-published-branches next-gen-html ${parameter}, whereas calling each individually allows us to pass a commandline arg to next-gen-html